content/en/real_user_monitoring/guide/best-practices-for-rum-sampling.md

```diff
@@ -11,55 +11,80 @@ further_reading:
+
+### Changing the sampling rate in the Datadog RUM UI
+You can change the sampling rate is only supported for server-side sampling and can be accomplished by [modifying the retention rate][15] from the retention filters page.
 
-RUM ensures availability of data when user devices are offline. In low-network areas, or when the device battery is too low, all RUM events are first stored on the local device in batches. They are sent as soon as the network becomes available, and the battery is high enough to ensure the RUM SDK does not impact the end user's experience. If the network is not available while your application is in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
+During live outages, if a bug or incident occurs, you can increase sampling to collect 100% of your sessions to ensure nothing is missed, or you have more examples of a particular issue.
 
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: /real_user_monitoring/guide/sampling-browser-plans/#overview
-[2]: /real_user_monitoring/guide/sampling-browser-plans/#overview
-[3]: /real_user_monitoring/mobile_and_tv_monitoring/android/advanced_configuration/?tab=kotlin#initialization-parameters
-[4]: /real_user_monitoring/ios/advanced_configuration/?tab=swift#sample-rum-sessions
-[5]: /real_user_monitoring/reactnative/#initialize-the-library-with-application-context
+[1]: /real_user_monitoring/guide/understanding-the-rum-event-hierarchy/#sessions
+[2]: /real_user_monitoring/rum_without_limits/
+[3]: /real_user_monitoring/guide/sampling-browser-plans/#overview
+[4]: /real_user_monitoring/mobile_and_tv_monitoring/android/advanced_configuration/?tab=kotlin#initialization-parameters
+[5]: /real_user_monitoring/ios/advanced_configuration/?tab=swift#sample-rum-sessions
 [6]: /real_user_monitoring/mobile_and_tv_monitoring/flutter/setup/advanced_configuration/#sample-rum-sessions
-[7]: /real_user_monitoring/mobile_and_tv_monitoring/roku/setup/#initialize-the-library
-[8]: /help
-[9]: /real_user_monitoring/guide/understanding-the-rum-event-hierarchy/#sessions
+[7]: /real_user_monitoring/mobile_and_tv_monitoring/kotlin_multiplatform/setup/?tab=rum#sample-rum-sessions
+[8]: /real_user_monitoring/reactnative/#initialize-the-library-with-application-context
+[9]: /real_user_monitoring/mobile_and_tv_monitoring/roku/setup/#initialize-the-library
+[10]: /real_user_monitoring/mobile_and_tv_monitoring/unity/setup#sample-rum-sessions
+[11]: /data_security/data_retention_periods/
+[12]: /real_user_monitoring/rum_without_limits/retention_filters
+[13]: /real_user_monitoring/rum_without_limits/metrics
+[14]: /real_user_monitoring/rum_without_limits/retention_filters#best-practices
```

Comment:
```suggestion
[14]: /real_user_monitoring/guide/retention_filter_best_practices/
```

Reply:
This link leads to section that only links out to another page, so suggesting this update to reduce the number of links a user has to click